### PR TITLE
Creating getDefaultConfig method

### PR DESCRIPTION
**Pull request by corbanbrook**
This provides a new method `getDefaultConfig` and a wrapping component that takes this config `SequenceKit`

### getDefaultConfig(walletType: 'waas' | 'universal', config: CombinedWaasConfig | CombinedUniversalConfig)

This method takes 2 params, the first specifies the wallet type (waas or universal) and the second is a combined config object which allows you to specify KitConfig as well as Connector options for the specific wallet type. You can also pass in any additional wagmi options with the `wagmiConfig` prop

```js
const config = getDefaultConfig('waas', {
  projectAccessKey: '...',
  waasClientId: '...',
  wagmiConfig: {
    ssr: true
  }
  ...
})
```

```jsx
const App = () => {
   return (
     <SequenceKit config={config}>
       <MyApp/>
     </SequenceKit>
   )
}

### CODE REVIEW

**Review by SamueleA — DISMISSED** [review]
Looks good! But I think we can delete the commented code.

**Review by SamueleA — APPROVED** [review]

File: examples/next/src/config.ts [hunk 1]
```diff
@@ -121,3 +42,24 @@ export const kitConfig: KitConfig = {
     }
   ]
 }
+
+export const config = getDefaultConfig('waas', {
```
Comment (pkieltyka) [inline examples/next/src/config.ts:46]: what are other options here in the first argument to `getDefaultConfig` besides 'waas' ?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,34 @@ yarn add @0xsequence/kit wagmi ethers@6.13.0 viem 0xsequence @tanstack/react-que
 
 ### Setting up the Library
 
+#### The 'easy' way
+
+```js
```
Comment (pkieltyka) [inline README.md:36]: so much better!

File: README.md [hunk 3]
```diff
@@ -31,6 +31,34 @@ yarn add @0xsequence/kit wagmi ethers@6.13.0 viem 0xsequence @tanstack/react-que
...
+import Content from './components/Content'
+
+const projectAccessKey = '<your-project-access-key>'
+
+const config = {
+  projectAccessKey,
+  chainIds: [1, 137]
+  defaultChainId: 1
+  appName: 'Demo Dapp'
+}
```
Comment (pkieltyka) [inline README.md:48]: are there any other config options on this object too..? for example, like a wallet connect id / key..?

File: README.md [hunk 4]
```diff
@@ -31,6 +31,34 @@ yarn add @0xsequence/kit wagmi ethers@6.13.0 viem 0xsequence @tanstack/react-que
...
+#### The 'easy' way
+
+```js
+import { SequenceKit } from '@0xsequence/kit'
+
+import Content from './components/Content'
+
+const projectAccessKey = '<your-project-access-key>'
+
+const config = {
```
Comment (pkieltyka) [inline README.md:43]: does `config` have a type..? maybe show, `const config: SequenceKitConfig = {` ?

File: examples/next/src/config.ts (hunk 1 above)
Comment (pkieltyka) [reply examples/next/src/config.ts:46]: ah I see now, its either 'waas' or 'universal'

could you update to `'embedded' | 'universal'` ? I think that will be more clear and more for this particular setting

Comment (corbanbrook) [reply examples/next/src/config.ts:46]: 'universal'

File: README.md (hunk 3 above)
Comment (corbanbrook) [reply README.md:48]: yes, i need to expannd the docs.  you can look at the next example config.ts

File: README.md (hunk 4 above)
Comment (corbanbrook) [reply README.md:43]: oops, that is actually `const config = getDefaultConfig(...)`

File: README.md [hunk 5]
```diff
@@ -31,6 +31,33 @@ yarn add @0xsequence/kit wagmi ethers@6.13.0 viem 0xsequence @tanstack/react-que
...
+#### The 'easy' way
+
+```js
+import { SequenceKit, getDefaultConfig } from '@0xsequence/kit'
+
+import Content from './components/Content'
+
+const projectAccessKey = '<your-project-access-key>'
+
+const config = getDefaultConfig('waas', {
```
Comment (pkieltyka) [inline README.md:43]: Maybe instead of the name “getDefaultConfig” we call it, “sequenceKitConfig” ..?

**Comment by tolgahan-arikan** [comment]
maybe we can use `embedded` instead of `waas` for the wallet type? because we are using that in docs etc. instead of `waas`

**Review by pkieltyka — APPROVED** [review]
LGTM :D

**Review by pkieltyka — APPROVED** [review]

**Pull request merged**